docs/sentry.rst

```diff
@@ -0,0 +1,73 @@
+.. highlight:: python
+
+
+======
+Sentry
+======
+
+Talisker provides out-of-the-box integration with sentry.
+
+Specifically, talisker adds:
+
+ * some default configuation of the sentry client
+ * sentry wsgi middleware
+ * sentry error log handler (for logged exception messages)
+ * log messge breadcrumbs (more breadcrumbs is a TODO)
+ * sentry integration with flask, django, and celery
+
+To get the current sentry client, simply use::
+
+    talisker.sentry.get_client()
+
+Error Data
+----------
+
+Talisker configures sentry breadcrumbs for log messages at INFO or higher level.
+It also addes the request id as a tag to the sentry message.
+
+If you want to add some custom error context, you can use the client above as you
+would use the sentry client as normal.
+
+e.g.::
```

Comment:
I'd collapse this into the previous paragraph, or alternatively use 'For example::' instead